attune/io/Topas4.py

```diff
@@ -0,0 +1,79 @@
+from .. import Instrument
+from .. import Tune
+from .. import Arrangement
+from typing import Dict, Union
+import json
+import numpy as np
+
+
+def fromTopas4(Topasfile):
+    # converts a LightConversion marked up Topas4 calibration JSON file into
+    # an Instrument object
+    f = open(Topasfile, "r")
+    lines = f.read()
+    f.close()
+    jsond = json.loads(lines)
```

Comment:
```suggestion
    with open(Topasfile, "r") as f:
        jsond = json.load(f)
```

With block is the recommended way of interacting with files, as it ensures the file is closed even if an exception occurs.

the json stdlib can read file objects directly, no need to read into a string first.